firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -69,30 +69,30 @@ void homeAxis(int motor[])
   currentMotorNumEighthSteps *= -1;
```

Comment:
For currentMotorNumEighthSteps and the other variables that do the same thing, Nashir and I decided on just assigning values to them. In this case, we'll be just setting them equal to the correct value, avoiding the multiplication.